pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -231,6 +235,33 @@ func (k *KubeadmBootstrapper) SetupCerts(k8s bootstrapper.KubernetesConfig) erro
 	return bootstrapper.SetupCerts(k.c, k8s)
 }
 
+func NewKubeletConfig(k8s bootstrapper.KubernetesConfig) (string, error) {
+	flags := []string{}
+	for _, opt := range k8s.ExtraOptions {
+		if opt.Component == "kubelet" {
+			flags = append(flags, fmt.Sprintf("--%s=%s", opt.Key, opt.Value))
+		}
+	}
+	// Strip leading 'v' prefix from version for semver parsing
+	fmt.Println(k8s.KubernetesVersion)
+	version, err := semver.Make(k8s.KubernetesVersion[1:])
+	if err != nil {
+		return "", errors.Wrap(err, "parsing kubernetes version")
+	}
+
+	// versions >= 1.8.0-alpha.0 require the --fail-swap-on flag set to false
+	if version.GTE(version18) {
```

Comment:
yuck, is this because or ISO has swap on? Should we just turn it off?

Reply:
Going forward this might be more maintainable if we have a map of special cased flags to versions, but this should be OK for now. Maybe just leave a TODO about that.